packages/appsync-modelgen-plugin/src/__tests__/utils/process-index.test.ts

```diff
@@ -132,6 +132,68 @@ describe('processIndex', () => {
     ]);
   });
 
+  it('should generate a default name to @index directive', () => {
+    const model: CodeGenModel = {
+      directives: [
+        {
+          name: 'model',
+          arguments: {},
+        },
+      ],
+      name: 'testModel',
+      type: 'model',
+      fields: [
+        {
+          type: 'field',
+          isList: false,
+          isNullable: true,
+          name: 'connectionField',
+          directives: [
+            {
+              name: 'index',
+              arguments: {},
+            },
+          ],
+        },
+        {
+          type: 'field',
+          isList: false,
+          isNullable: true,
+          name: 'anotherConnection',
+          directives: [
+            {
+              name: 'index',
+              arguments: {},
```

Comment:
The `sortKeyFields` can be added in the argument to test a different scenario from the one above